spec/forms/idv/doc_pii_form_spec.rb

```diff
@@ -18,21 +18,28 @@
       zipcode: Faker::Address.zip_code,
       state: Faker::Address.state_abbr,
       state_id_jurisdiction: 'AL',
+      state_id_number: 'S59397998',
```

Comment:
Could you define `S59397998` at the top of the file
`let(:state_id_number) { 'S59397998' }` and then refer to `state_id_number` throughout the file?

Reply:
I can do that. I feel like with tests you're supposed to prefer defining everything inside of each test? 🤷

Reply:
The more I look at this, the more I don't think that it makes sense to make the change. The way I've done this follows the "pattern" that each test uses and changing it wouldn't solve anything, but might add confusion.

Reply:
... i understand following the pattern that's set. I do think the testing could benefit from defining fields a the top and not just state_id_number. i think it'd be more readable/manageable if we moved in that direction . overall .... i think this PR looks great ... LGTM

Reply:
Thanks for the feedback Amir and Eileen! Moved the :state_id_number up to the top of the file ([commit](https://github.com/18F/identity-idp/pull/9777/commits/7ca30397ab7df0b2e0f5969cdb15b1d364294b58)).